src/index.ts

```diff
@@ -20,4 +20,5 @@ export function provideAnalytics(config: AnalyticsConfig): AnalyticsService {
 }
 
 export * from './models';
-export { AnalyticsService };
\ No newline at end of file
+export * from './services';
+export { AnalyticsReporter };
```

Comment:
Do we need to export `AnalyticsReporter`? I thought `AnalyticsService` would be sufficient?

Reply:
we no longer need to export it since it's now internal